tests/encodeinput.cpp

```diff
@@ -176,29 +177,54 @@ EncodeOutput* EncodeOutput::create(const char* outputFileName, int width , int h
     if(ext==NULL)
         return NULL;
     ext++;//h264;264;jsv;avc;26l;jvt;ivf
-    if(strcasecmp(ext,"h264")==0 ||
-        strcasecmp(ext,"264")==0 ||
-        strcasecmp(ext,"jsv")==0 ||
-        strcasecmp(ext,"avc")==0 ||
-        strcasecmp(ext,"26l")==0 ||
-        strcasecmp(ext,"jvt")==0 ) {
+
+    if (!strcasecmp(codecName, "AVC")) {
```

Comment:
Actually, this command reserved for .mkv or .mp4. Simpler solution for this is make .vp9 stand for vp9.  vp8 and ivf stand for vp8. :).   It is just our demo encoder; we can choose the extension and means. However, once you decide use the param, you had better make it consistency.